### DISCUSSION TITLE
[ChatGPT 咒語庫] Midjourney 咒語產生器 | 全端開發人員天梯

### DISCUSSION THREAD

**Discussion by giscus** [wellwind/wellwind.github.io#80] [Announcements]
# [ChatGPT 咒語庫] Midjourney 咒語產生器 | 全端開發人員天梯

Midjourney 應該算是當前最熱門的 AI 繪圖軟體之一了！只要把我們想像的文字丟進去給 Midjourney，就能產出非常精美的圖片，雖然細節有時候還是會有問題 (例如經典的手指頭…)，但至少對於我這種美術白癡來說，已經是非常好看了！不過要能很好的跟 Midjourney 詠唱咒語，難度可是比 ChatGPT 高得多了，但也別低估 ChatGPT，我們可以透過 ChatGPT 幫助我們補強細節，讓 Midjourney 產生出更好的圖片！

https://fullstackladder.dev/blog/2023/02/13/chat-gpt-prompts-midjourney-generator/

**Comment by b3326023** [comment]
厲害，感謝分享!
請教一下，我輸入了文章中的咒語後，成功將chatGPT變成一個產生器，但是為什麼他自己就開始寫 Midjourney了，我都還沒提供任何繪圖的 idea 給他耶

**Reply by wellwind** [reply]
Hi，整段咒語中間有一段

```txt
[1] = [[實際描述]]
```

這裡記得要把 `[[實際描述]]` 換掉喔！

**Comment by besthand** [comment]
Hello Mike：

我是《硬是要學》網站編輯，我們正在收集實用的 ChatGPT 指令，不曉得是否能授權我們將您提供的指令蒐集到我們製作的 ChatGPT 指令大補帖 (https://www.soft4fun.net/prompt-type/knowledge) 與網友分享呢？ 

謝謝！

**Reply by wellwind** [reply]
您好，完全沒有問題喔！感謝您
